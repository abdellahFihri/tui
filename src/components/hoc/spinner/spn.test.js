import React from 'react';
import {shallow} from 'enzyme'
import CircularUnderLoad from './spinner'

it('expect to render spinner component',()=>{

    expect(shallow(<CircularUnderLoad/>)).toMatchSnapshot();
})