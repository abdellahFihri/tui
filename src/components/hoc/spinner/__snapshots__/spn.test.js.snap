// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render spinner component 1`] = `ShallowWrapper {}`;
